File: skills/astra-toolkit/clients/java/examples/tables-drop-type-exists.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;
import com.datastax.astra.client.tables.commands.options.DropTypeOptions;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN")).getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"));

    // Drop a user-defined type
    DropTypeOptions options = new DropTypeOptions().ifExists(true);
    database.dropType("**UDT_NAME**", options);
  }
}

File: skills/astra-toolkit/clients/typescript/examples/tables-drop-type-exists.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});

// Drop a user-defined type
(async function () {
  await database.dropType("**UDT_NAME**", { ifExists: true });
})();
